9.2/run-postgresql.sh

```diff
@@ -56,12 +67,8 @@ if [ "$1" = "postgres" -a ! -f "$PGDATA/postgresql.conf" ]; then
 	# PostgreSQL configuration.
```

Comment:
Pls say here that this will be **appended** to the postgresql.conf, allowing users to modify it as they want.